### DISCUSSION TITLE
Is it possible to only install material-react-table and create a headless table using hooks?

### DISCUSSION THREAD

**Discussion by emadabdulrahim** [KevinVandy/material-react-table#1128] [Q&A] [answered]
I don't use any @mui package, but I like MRT since I already use tanstack table. I want to build my own UI using `useMRT_TableOptions` and `useMRT_TableInstance`. However, I get lots of errors that indicate missing component packages.

Why is it required that I install `@mui/material` if I'm only interested in MRT hooks?

Is there a way to achieve what I want?

**Comment by KevinVandy** [comment, marked as answer]
You should just use TanStack Table.

**Reply by emadabdulrahim** [reply]
Cool, thanks!
